tests/e2e/queuePopulator/config/s3c-config.json

```diff
@@ -0,0 +1,171 @@
+{
+  "zookeeper": {
+    "connectionString": "127.0.0.1:2181/backbeat",
```

Comment:
this is used from backbeat running inside a container?
- if it is, then the connectionString should be `zookeeper:2181`
- if it runs on the host, then indeed may be something like this... (ideally the port should be dynamically allocated, but that can be changed later)

Reply:
This config is used in the tests and not inside a container (we initialize the queuePopulator in the tests with this).